Cargo.lock

```diff
@@ -5490,7 +5502,20 @@ dependencies = [
  "termcolor",
  "tracing",
  "tracing-log",
- "tracing-subscriber",
+ "tracing-subscriber 0.2.16",
+]
+
+[[package]]
+name = "tracing-tree"
+version = "0.2.0"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "3ce989c9962c7f61fe084dd4a230eec784649dfc2392467c790007c3a6e134e7"
+dependencies = [
+ "ansi_term 0.12.1",
+ "atty",
+ "tracing-core",
+ "tracing-log",
+ "tracing-subscriber 0.3.3",
 ]
```

Comment:
Since we probably don't want to pull in this dependency.

Reply:
I [disabled](https://github.com/pierwill/rust/blob/4b37963c86e66b33825212549127924a9a9e5596/compiler/rustc_traits/Cargo.toml#L17) defaulted feature for chalk-solve. How do I make sure this is removed from the lock file?

Reply:
Uh...hmm. I guess other crates pull in tracing-tree, but 0.1.9. If you reset the lock file to master and rebuild, maybe that'll work?

Reply:
I did

```
$ git co master Cargo.lock
$ ./x.py
```

tracing-tree 0.2.0 is still pulled into the lock file. :(